bin/main.rb

```diff
@@ -1,2 +1,30 @@
-#!/usr/bin/env ruby
-puts 'Hello world!'
```

Comment:
- [ ] **Hint1:** You deleted your environment declaration, that is why the program is not being executed correctly with `bin/main.rb`

Reply:
DONE